services/httpd/pprof.go

```diff
@@ -113,15 +116,15 @@ func (h *Handler) archiveProfilesAndQueries(w http.ResponseWriter, r *http.Reque
 				return
 			}
 
-			if err := prof.WriteTo(&buf, int(profile.Debug)); err != nil {
+			if err := prof.WriteTo(&buf, 0); err != nil {
 				http.Error(w, err.Error(), http.StatusInternalServerError)
 				return
 			}
 		}
 
 		// Write the profile file's header.
 		err := tw.WriteHeader(&tar.Header{
-			Name: profile.Name + ".txt",
+			Name: path.Join("profiles", profile.Name+".pb.gz"),
```

Comment:
Since we're now instructing the profiler to generate the modern format (non-legacy), which is gzipped, does this `tw` still need to wrap a `gzip.Writer` on line 99 above?

Reply:
ya -- we could probably not gzip the entire tarball.

Reply:
done!